cmd/openshift-install/create.go

```diff
@@ -197,9 +198,59 @@ func runTargetCmd(targets ...asset.WritableAsset) func(cmd *cobra.Command, args
 		if err != nil {
 			logrus.Fatal(err)
 		}
+		logFilesChanged(cmd.Name(), rootOpts.dir, targets)
 	}
 }
 
+// logFilesChanged checks all the asset files created and logs it for the user to see
+func logFilesChanged(cmdName string, directory string, targets []asset.WritableAsset) {
```

Comment:
move it out to a pkg (i don't think we should put this code in main package), and try adding some unit tests.